eng/common/scripts/TypeSpec-Project-Process.ps1

```diff
@@ -124,8 +124,20 @@ if ($TypeSpecProjectDirectory -match '^https://github.com/(?<repo>Azure/azure-re
     Write-Error "Failed to find tspconfig.yaml in '$TypeSpecProjectDirectory'"
     exit 1
   }
+  if($TypeSpecProjectDirectory -match "^.*/(?<path>specification/.*)$") {
```

Comment:
We should move to before line 122 where you are using it to find the tspconfig location. If this path isn't valid we should know it there.

Reply:
Updated.